docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md

```diff
@@ -129,48 +130,56 @@ When using OIDC, set `connectors.inbound.mode: disabled` [in your Connectors Hel
 
 ```
    CAMUNDA_IDENTITY_TYPE=MICROSOFT
-   CAMUNDA_IDENTITY_ISSUER=<URL_OF_ISSUER>
-   CAMUNDA_IDENTITY_ISSUER_BACKEND_URL=<URL_OF_ISSUER> // this is used for container to container communication
+   CAMUNDA_IDENTITY_ISSUER=https://login.microsoftonline.com/<Client ID from step 1>/v2.0
```

Comment:
🔧 this should be the ID of the tenant in Entra instead of the Client ID

Reply:
```suggestion
   CAMUNDA_IDENTITY_ISSUER=https://login.microsoftonline.com/<Microsoft Entra tenant id>/v2.0
```

Or should this just be `<Microsoft Entra tenant id>`?

Reply:
This looks good to me!